### PR TITLE
[tabulator-tables] Updates from 5.4

### PR DESCRIPTION
**Pull request by Jojoshua**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes:(http://tabulator.info/docs/5.4/release)
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/tabulator-tables/tabulator-tables-tests.ts [hunk 1]
```diff
@@ -1126,36 +1126,67 @@ table = new Tabulator('#test', {
...
+            headerMenu: headerMenuForIconTest,
+        },
+    ],
 });
+
+// 5.4
+
+table = new Tabulator('#test', {
+    headerSortClickElement: 'icon',
+    groupDblClickPopup: 'Im a Popup',
```
Comment (mike-lischke) [inline types/tabulator-tables/tabulator-tables-tests.ts:1167]: Maybe you want to fix the typo in all occurrences of this string? "I'm a Popup"...

Comment (Jojoshua) [reply types/tabulator-tables/tabulator-tables-tests.ts:1167]: Eh, not really for the tests. Do you feel strongly about it?

Comment (mike-lischke) [reply types/tabulator-tables/tabulator-tables-tests.ts:1167]: It's a simple change and makes it look better if written correctly. You can also use [Lorem Ipsum](https://en.wikipedia.org/wiki/Lorem_ipsum) for dummy text.

I don't feel strongly about that, but I look after the small things too. If you say "let's keep that" then so be it and I'll approve.

Comment (Jojoshua) [reply types/tabulator-tables/tabulator-tables-tests.ts:1167]: It was copied from the website examples again. I kind of do want to fix it but don't want to spend the time this morning to bring it back up, I'm on mobile this morning . Would you want to push the update?

**Review by mike-lischke — APPROVED** [review]

**Comment by Jojoshua** [comment]
Ready to merge

**Pull request merged**